config/opal_check_package.m4

```diff
@@ -208,7 +205,7 @@ AC_DEFUN([OPAL_CHECK_PACKAGE],[
                 [opal_check_package_happy="yes"],
                 [opal_check_package_happy="no"])],
           [opal_check_package_happy="no"],
-          [$10])
+          [])
```

Comment:
Did you check that in all places that we call `OPAL_CHECK_PACKAGE` from that we don't need any extra includes?

Reply:
Yep, i did check that.

Reply:
Cool.  It is safe to remove the `includes` param from `OPAL_CHECK_PACKAGE_HEADER()`, then?  Or do other places call it that need `includes`?